### DISCUSSION TITLE
Add Ability to add Individual Issues to Collections and also Collections to Collections.

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#2751] [Ideas]
### Idea Description

1. Individual issues to collections.
This will allow creating collections of artists or authors and is very important when creating these out of very large comic volumes. Currently, the only thing searching for these sort of queries is to find the volume. But if there are over 100 issues and the artist in question only did a few issues, it renders searching for them in the first place moot.

2. Ability to add collections to collections will allow for creating more general collections of specific collections. For example, I have 7 genres of comics, but I want 1 collection with all of those genres together.

### Idea Category

User Experience

### Duration of Using Kavita

1 week but long experience using media servers.

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by maximo1984** [comment]
I see, that if you add a series of issues to a reading list, you can then add those single issues to a collection. While this does work, it is an extra step. This feature should be easily added to browsing issues from where they first appear on import.

**Comment by majora2007** [comment]
Collections in most software are always done at a series level while reading list is done at an individual issue level. I don't have any plans in changing collections when you could instead just use a reading list with an arbitrary order (or imply your own based on publication status). 

For collection in collections, there is an existing issue that you can upvote.
